server/routes/activities/manage-allocations/handlers/allocateMultiplePeople/uploadPrisonerList.test.ts

```diff
@@ -226,74 +226,6 @@ describe('Allocate multiple people to an activity - upload a prisoner list', ()
       expect(req.session.allocateJourney.inmates).toEqual(expectedInmates)
       expect(res.redirect).toHaveBeenCalledWith('review-upload-prisoner-list')
     })
-
```

Comment:
I think it might be a good idea if we test the new code that you've put into uploadPrisonerList.ts - maybe check that  notFoundPrisoners has been added to the session, and unidentifiable, when they're meant to have been?